frontend/lib/identification/activation_workflow/activation_code_scanner_page.dart

```diff
@@ -97,12 +98,13 @@ class ActivationCodeScannerPage extends StatelessWidget {
           throw const ActivationInvalidTotpSecretException();
         }
         final totpSecret = const Base64Decoder().convert(activationResult.totpSecret!);
-        final userCode = DynamicUserCode(
-          info: activationCode.info,
-          pepper: activationCode.pepper,
-          totpSecret: totpSecret,
-        );
-        provider.setCode(userCode);
+
+        provider.setCode(DynamicUserCode(
+            info: activationCode.info,
+            pepper: activationCode.pepper,
+            totpSecret: totpSecret,
+            cardVerification:
+                CardVerification(cardValid: true, verificationTimeStamp: daysSinceEpoch(DateTime.now().toUtc()))));
```

Comment:
We should optimally also use a backend created timestamp here.
The good thing here: We would not break the API, if we add a field `activationTimeStamp` to `activationResult`.